docs/community_resources/contributing.md

```diff
@@ -222,23 +224,23 @@ As stated in the HTTP section, all unsafe requests to the web server should have
 
 All data that comes from backend and in return comes from user input should always be escaped, in case of Twig templates this done by default, but in case of PHP templates, Ajax and other not Twig based output this must be handled manually.
 
-Output escaping must be properly executed according to the desired format, eg. javascript vs. html, but also taking into account the correct character set (see eg. output escaping fallacy when not specifying charset encoding in [htmlspecialchars](http://www.php.net/htmlspecialchars))
+Output escaping must be properly executed according to the desired format, eg. javascript vs. html, but also taking into account the correct character set (see e.g. output escaping fallacy when not specifying charset encoding in [htmlspecialchars](http://www.php.net/htmlspecialchars))
 
 #### Admin
 
-Admin operations that can have a severe impact on the web applications should require providing password and require it again after some time has gone, normally 10 - 20 minutes, on all session based interfaces.
+Admin operations that can have a severe impact on the web applications should require providing password and require it again after some time has gone, normally 10-20 minutes, on all session based interfaces.
 
 ### PHP
 
-For now see our comprehensive coding standard & guidelines [wiki page](https://github.com/ezsystems/ezpublish-kernel/wiki/codingstandards) on github.
+For now see our comprehensive coding standard and guidelines [wiki page](https://github.com/ezsystems/ezpublish-kernel/wiki/codingstandards) on github.
```

Comment:
The link does not work, I'm not sure the page exists anyway. Best to ask around devs if we have any equivalent to this at the moment.